src/test/java/spoon/test/targeted/TargetedExpressionTest.java

```diff
@@ -265,7 +252,7 @@ public void testStaticTargetsOfFieldAccessNoClasspath() throws Exception {
 		assertEqualsFieldAccess(new ExpectedTargetedExpression().declaringType(expectedFoo).target(exepectedThisAccess).result("foo"), elements.get(2));
 		assertEqualsFieldAccess(new ExpectedTargetedExpression().declaringType(expectedFoo).target(exepectedThisAccess).result("Foo.this.i"), elements.get(3));
 		assertEqualsFieldAccess(new ExpectedTargetedExpression().declaringType(expectedFoo).target(exepectedThisAccess).result("foo"), elements.get(4));
-		assertEqualsFieldAccess(new ExpectedTargetedExpression().declaringType(expectedFoo).target(expectedTypeAccess).result("Foo.staticField"), elements.get(5));
+		assertEqualsFieldAccess(new ExpectedTargetedExpression().declaringType(expectedFoo).target(expectedTypeAccess.toString()).result("Foo.staticField"), elements.get(5));
```

Comment:
do you mean that after the refactoring, a CtSnippet will not be equal
with an object that represents the same code? but currently it works?
(it makes sense)